src/test/java/patio/infrastructure/graphql/fetchers/VotingStatsFetcherTests.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (C) 2019 Kaleidos Open Source SL
+ *
+ * This file is part of Don't Worry Be Happy (DWBH).
+ * DWBH is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * DWBH is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with DWBH.  If not, see <https://www.gnu.org/licenses/>
+ */
+package patio.infrastructure.graphql.fetchers;
+
+import static io.github.benas.randombeans.api.EnhancedRandom.random;
+import static org.junit.Assert.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+
+import java.time.OffsetDateTime;
+import java.util.HashMap;
+import java.util.List;
+import java.util.UUID;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import patio.common.domain.utils.PaginationResult;
+import patio.infrastructure.graphql.fetchers.utils.FetcherTestUtils;
+import patio.user.domain.User;
+import patio.voting.domain.VotingStats;
+import patio.voting.graphql.VotingFetcher;
+import patio.voting.graphql.VotingStatsFetcher;
+import patio.voting.services.internal.DefaultVotingStatsService;
+
+/**
+ * Tests {@link VotingFetcher} class
+ *
+ * @since 0.1.0
+ */
+class VotingStatsFetcherTests {
+
+  @Test
+  void testCreateVoting() {
```

Comment:
Is this test checking `voting creation` ? I'm saying this because this test is named `testCreateVoting`. I think it should be named something else.

Reply:
Yes, for sure. I'll change it